src/vue-components/popup.vue

```diff
@@ -474,8 +477,33 @@
       dragMoveListener (event) {
         if (this.draggable) {
           const target = event.target;
-          const x = (parseFloat(target.getAttribute('data-x')) || 0) + event.dx;
-          const y = (parseFloat(target.getAttribute('data-y')) || 0) + event.dy;
+          let dx = event.dx
+          let dy = event.dy
+
+          /*
+          On some websites Interact.js is unable to determine correct clientX or clientY coordinates.
+          This will result in a popup moving abruptly beyond screen limits.
+          To fix this, we will filter out erroneous coordinates and chancel a move in the corresponding
+          direction as incorrect. This will allow us to keep the popup on screen by sacrificing its movement
+          in (usually) one direction. This is probably the best we can do with all the information we have.
+           */
+          const drugTreshold = 100 // Drag distance values above this will be considered abnormal
```

Comment:
this is really nitpicky, but probably should name this dragThreshold not drugThreshold

Reply:
Oh, that's right, thanks for catching this! Will correct.